hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/ContainerStateMachine.java

```diff
@@ -581,7 +581,10 @@ private CompletableFuture<Message> writeStateMachineData(
     writeChunkFuture.thenApply(r -> {
       if (r.getResult() != ContainerProtos.Result.SUCCESS
           && r.getResult() != ContainerProtos.Result.CONTAINER_NOT_OPEN
-          && r.getResult() != ContainerProtos.Result.CLOSED_CONTAINER_IO) {
+          && r.getResult() != ContainerProtos.Result.CLOSED_CONTAINER_IO
+          // After concurrent flushes are allowed on the same key, chunk file inconsistencies can happen and
```

Comment:
is this being fixed by [HDDS-11328](https://issues.apache.org/jira/browse/HDDS-11328)?

Reply:
Turns out it's a simple fix so I include it here. I can move it to a separate PR if neccessary.

Reply:
Got it. This is fine. Thanks.